postgres/rdspostgres/rdspostgres_test.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package rdspostgres
+
+import (
+	"context"
+	"testing"
+
+	"github.com/google/go-cloud/aws/rds"
+	"github.com/google/go-cloud/internal/testing/terraform"
+)
+
+func TestOpen(t *testing.T) {
```

Comment:
I'm not so sure about default zero values, since the `Open` function mostly just passes the values along to the underlying driver. I did add a test for the improper extra parameters you pointed out in #737. WDYT?